### DISCUSSION TITLE
How to Set "Received: from" Header to Mail Server Address Instead of Console Domain?

### DISCUSSION THREAD

**Discussion by emahdij** [postalserver/postal#3087] [Help with using Postal] [answered]
Hi everyone,

I've noticed that when I send emails, the "Received: from" header is being set to the console domain (the domain where Postal is hosted). However, I would like this header to reflect the actual mail server address instead.

Is there a way to configure Postal so that the "Received: from" header shows the mail server address instead of the console domain? Any advice or configuration tips would be greatly appreciated!

Thanks in advance for your help!

Best regards,
Mehdi

**Comment by willpower232** [comment]
Do you mean the header in the email or the one which appeared at the top of the email client?

**Reply by emahdij** [reply]
I think the first one.
I saw it in this benchmark:
![telegram-cloud-photo-size-4-5900143853694993637-y](https://github.com/user-attachments/assets/4042f2ae-97bc-428e-8607-470003ac652c)
The first line shows the sender's IP or Domain. That is this example is my postal console domain.
Deliverability Test:
https://mxtoolbox.com/deliverability

**Reply by willpower232** [reply, marked as answer]
I think  you can hide most of the information with privacy mode 

![image](https://github.com/user-attachments/assets/b2bd33b5-837d-4779-bcbf-c83239f727d3)
